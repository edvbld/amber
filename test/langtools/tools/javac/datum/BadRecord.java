/*
 * @test /nodynamiccopyright/
 * @summary Verifying error recovery for broken record classes
 * @compile/fail/ref=BadRecord.out -XDrawDiagnostics BadRecord.java
 */
record BadRecordb1 {}

record BadRecordb2( {}

record BadRecordb3(int {}

record BadRecordb4(int) {}

record BadRecordb5(int i {}

record BadRecordb6(int i, {}

record BadRecordb7(int i,) {}

record BadRecordb8(int i, int {}

record BadRecordb9(int i, int) {}

record BadRecordba(int i, int j {}

record BadRecordbb(int i, int j, {}

record BadRecordbc(int i, int j,) {}

record BadRecords1;

record BadRecords2(;

record BadRecords3(int;

record BadRecords4(int);

record BadRecords5(int i;

record BadRecords6(int i,;

record BadRecords7(int i,);

record BadRecords8(int i, int;

record BadRecords9(int i, int);

record BadRecordsa(int i, int j;

record BadRecordsb(int i, int j,;

record BadRecordsc(int i, int j,);
