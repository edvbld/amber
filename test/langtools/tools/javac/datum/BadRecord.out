BadRecord.java:6:19: compiler.err.expected: '('
BadRecord.java:8:21: compiler.err.illegal.start.of.type
BadRecord.java:10:23: compiler.err.expected: token.identifier
BadRecord.java:12:23: compiler.err.expected: token.identifier
BadRecord.java:14:25: compiler.err.expected: ')'
BadRecord.java:16:27: compiler.err.illegal.start.of.type
BadRecord.java:18:26: compiler.err.illegal.start.of.type
BadRecord.java:20:30: compiler.err.expected: token.identifier
BadRecord.java:22:30: compiler.err.expected: token.identifier
BadRecord.java:24:32: compiler.err.expected: ')'
BadRecord.java:26:34: compiler.err.illegal.start.of.type
BadRecord.java:28:33: compiler.err.illegal.start.of.type
BadRecord.java:30:19: compiler.err.expected: '('
BadRecord.java:32:20: compiler.err.illegal.start.of.type
BadRecord.java:34:23: compiler.err.expected: token.identifier
BadRecord.java:36:23: compiler.err.expected: token.identifier
BadRecord.java:38:25: compiler.err.expected: ')'
BadRecord.java:40:26: compiler.err.illegal.start.of.type
BadRecord.java:42:26: compiler.err.illegal.start.of.type
BadRecord.java:44:30: compiler.err.expected: token.identifier
BadRecord.java:46:30: compiler.err.expected: token.identifier
BadRecord.java:48:32: compiler.err.expected: ')'
BadRecord.java:50:33: compiler.err.illegal.start.of.type
BadRecord.java:52:33: compiler.err.illegal.start.of.type
24 errors
